ironfish-cli/src/utils/currency.ts

```diff
@@ -59,6 +59,7 @@ export async function promptCurrency(options: {
     Assert.isNotNull(amount)
 
     if (options.minimum != null && amount < options.minimum) {
+      console.log("Please enter an amount greater than 0")
```

Comment:
This message won't always be correct: what if `options.minimum` isn't equal to 1?

We also prefer not to use `console.log`.

I think that it might be better to update `text` to include the minimum (if any), similar to what we do for the balance on line 40.

Reply:
Yes we shouldn't use console.log. You need to pass a logger in, and use that.

I'm ok if we don't update the text, I did that in my original Pr and it felt like too much information for an error case that most users won't hit.

Reply:
Perfect! I used a logger, tested it, and pushed a new commit